test/connectivity/modules/NetworkController/NetworkProfile.cs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace NetworkController
+{
+    public class NetworkProfile
+    {
+        public string ProfileType { get; set; }
+
+        public NetworkProfileSetting ProfileSetting { get; set; }
```

Comment:
if NetworkProfile and NetworkProfileSetting are 1-to-1 mapping, why just combine it? move all properties from NetworkProfileSetting to NetworkProfile.